server/tests/controller/signup.spec.js

```diff
@@ -0,0 +1,149 @@
+import chai from "chai";
+import uniqid from "uniqid";
+import chaiHttp from "chai-http";
+import app from "../../index";
+import database from "../../models/index";
+import userData from "../fixtures/models/userData";
+import { generateJWT, decodeJWT, getJWTConfigs } from "../../helpers/jwt";
+import { getBaseUrl } from "../../helpers/userHelpers";
+
+chai.use(chaiHttp);
+const { expect, assert } = chai;
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true }).catch(() => {});
+});
+const userDependencies = async () => {
+  const user = await User.create({
+    firstName,
+    lastName,
+    password,
```

Comment:
This should be in helper in the tests in case somebody else wants to use a test with a created user